pss/pss_test.go

```diff
@@ -1777,6 +1794,65 @@ func benchmarkSymkeyBruteforceSameaddr(b *testing.B) {
 	}
 }
 
+func BenchmarkMessageProcessing(b *testing.B) {
+	b.Run(fmt.Sprintf("FailProb%.2f", 0.0), func(b *testing.B) { benchmarkMessageProcessing(b, 0.0) })
+	b.Run(fmt.Sprintf("FailProb%.2f", 0.01), func(b *testing.B) { benchmarkMessageProcessing(b, 0.01) })
+	b.Run(fmt.Sprintf("FailProb%.2f", 0.05), func(b *testing.B) { benchmarkMessageProcessing(b, 0.05) })
+}
+
+func benchmarkMessageProcessing(b *testing.B, failProb float32) {
+	rand.Seed(0)
```

Comment:
Please add the `b.N` loop

Reply:
That could be the reason for the strange results

Reply:
Ok, added. Now it seems more stable

Reply:
this benchmark is myterious, please rethink :)

Reply:
FIxed in 1b91b66. Now it is more stable with these values
```bash
$ go test -v -bench=BenchmarkMessageProcessing -run=^$
goos: linux
goarch: amd64
pkg: github.com/ethersphere/swarm/pss
BenchmarkMessageProcessing/FailProb0.00-4                      1        2915534902 ns/op        168677016 B/op   1752294 allocs/op
BenchmarkMessageProcessing/FailProb0.01-4                      1        1072452793 ns/op        92210520 B/op    1668221 allocs/op
BenchmarkMessageProcessing/FailProb0.05-4                      2         758722005 ns/op        89287224 B/op    1703359 allocs/op
PASS
ok      github.com/ethersphere/swarm/pss        6.974s

```

Reply:
@zelig comments like that aren't helpful. If you have objections then please share them concisely.